### PR TITLE
Apply monochrome icon

### LINKED ISSUE #755: Add Monochromatic Icon
**Idea Description**
For Android 13 (API level 33), we can use Monochromatic Icon. Please apply it!

**Reference images and links**
https://developer.android.com/develop/ui/views/launch/icon_design_adaptive#user-theming
https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?type=design&node-id=57713-28823&mode=design&t=iH9C6vVMHbFFKiMf-0
image

### PR DESCRIPTION
**Pull request by Dai1678**
## Issue
- close #755 

## Overview (Required)
- Added `<monochrome>` elements

## Links
- https://developer.android.com/develop/ui/views/launch/icon_design_adaptive#add_your_adaptive_icon_to_your_app

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app-android/src/main/res/mipmap-anydpi-v26/ic_launcher.xml
Comment (Dai1678) [inline app-android/src/main/res/mipmap-anydpi-v26/ic_launcher.xml:5]: At first, I downloaded [svg image](https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?type=design&node-id=57713-28823&mode=design&t=iH9C6vVMHbFFKiMf-0) from Figma.
But, However, the output XML data was quite different from the existing ic_launcher_foreground.xml, so the icon size became larger than expected.

If you are fine with the results shown in the screenshot, we can reuse ic_launcher_foreground.
This is based on the response last year. https://github.com/DroidKaigi/conference-app-2022/pull/321

**Comment by Dai1678** [comment]
@takahirom 
Excuse me, does the workflow require approval from the maintainer to run?

**Review by momomomo111 — APPROVED** [review]
Thank you! I appreciate you incorporating the past solutions!"

File: app-android/src/main/res/mipmap-anydpi-v26/ic_launcher.xml
Comment (takahirom) [reply app-android/src/main/res/mipmap-anydpi-v26/ic_launcher.xml:5]: Let me consult with the designer

**Comment by takahirom** [comment]
I've talked to the designer! Thanks! Thank you for finding the way to do this!

**Pull request merged**
